### ISSUE TITLE
I can't open an Asset in the Inspector mode

### ISSUE THREAD

**Issue by jdim** [contentful/live-preview#384]
Hello. Does the inspector mode support select images?

When I click the `Edit` button on the image, the left panel can't open the field and shows "Entry missing or inaccessible".

It seems it is because the URL points to `/entries/<id>`
e.g.
https://api.contentful.com/spaces/ba61t2k72fba/entries/5NiLieply8fnzpgmTT0mGS

but shoud be `/assets/<id>`
https://api.contentful.com/spaces/ba61t2k72fba/assets/5NiLieply8fnzpgmTT0mGS

So I got a 404

Thanks.

<img width="1512" alt="Screenshot 2023-10-27 at 18 05 43" src="https://github.com/contentful/live-preview/assets/8625641/6e128759-c49e-4d11-8641-385e9418423e">

**Comment by YvesRijckaert** [comment]
Hey! For assets you can tag the asset field of an entry.
E.g. in my `Post` content type I have an `image` Media field. You can use the field id of the `image` field to tag your asset.

**Comment by jdim** [comment]
Thank you. Could you please help what am I missing?
For my example, I have a `Page C` content type. It has an `RImage` field with id `rImage`.
I use this field id as `fieldId` in `ContentfulLivePreview()`.

The content type:

<img width="1442" alt="image" src="https://github.com/contentful/live-preview/assets/8625641/b21c93c4-0643-41d8-9b7c-6a54fdca0bd3">

`RImage` media field:

<img width="1487" alt="image" src="https://github.com/contentful/live-preview/assets/8625641/3e24357f-bac1-4573-b22a-b9b6ddd5eccd">

I use `ContentfulLivePreview.getProps` function:

```typescript
"use client";

import Image from "next/image";
import { LivePreviewProps } from "../types/livePreview";
import { ContentfulLivePreview } from "@contentful/live-preview";

interface ContentfulImageProps {
  src: string;
  width?: number;
  quality?: number;
  [key: string]: any; // For other props that might be passed
}

const contentfulLoader = ({ src, width, quality }: ContentfulImageProps) => {
  return `${src}?w=${width}&q=${quality || 75}`;
};

export default function ContentfulImage(
  props: ContentfulImageProps & LivePreviewProps
) {
  const { entryId, fieldId, ...rest } = props;
  return (
    <Image
      alt={props.alt}
      loader={contentfulLoader}
      {...ContentfulLivePreview.getProps({ entryId, fieldId })}
      {...rest}
    />
  );
}

```

And I send these props for live preview 

```typescript
<ContentfulImage
  src="{rImage.url!}"
  height="{rImage.height}"
  width="{rImage.width!}"
  className="object-cover"
  alt=""
  // 👇👇👇
  fieldId="rImage"
  entryId="{rImage.sys.id}"
/>

```

**Comment by YvesRijckaert** [comment]
Can you check which id you are using? It should be the entry id not the asset id.

For example:
This entry id would be correct: `2DKcPfjV6sEOea0zUGObgg` `https://app.contentful.com/spaces/c586i03gpxi9/entries/2DKcPfjV6sEOea0zUGObgg`
Not the asset id: `3JECogfmlqLlXY93QJ2t2F` `https://app.contentful.com/spaces/c586i03gpxi9/assets/3JECogfmlqLlXY93QJ2t2F`

**Comment by jdim** [comment]
Thank you.
If I got it right, for media fields we should use the parent's entry id for the correct behavior of inspector mode.
For example, if I got an entry type `PageC` containing a media field with the name `rImage` for an image component in `ContentfulLivePreview.getProps()` I should use props fieldId="rImage" & entryId={pageEntry.sys.id} (NOT rImage.sys.id).

As a result: a click on the `Edit` button, does `scroll to image` on the entry page.


![ezgif-5-f9ebb2eeda](https://github.com/contentful/live-preview/assets/8625641/ae319788-8e75-4cfe-a23e-c2739218a585)
